ci/jenkins/jobs/job-templates.yaml

```diff
@@ -333,6 +333,64 @@
      publishers: '{publishers}'
      wrappers: '{wrappers}'
 
+- job-template:
+      name: 'cloud-{name}-{test_name}-for-pull-request'
```

Comment:
Is it necessary to introduce a new job-template. What's the difference between this job-template and the existing ones.

Reply:
Can you either use an existing job template, or explain why it's necessary to use a new job-template?